src/clients/dotnet/dotnet_bindings.zig

```diff
@@ -134,23 +134,21 @@ fn get_mapped_type_name(comptime Type: type) ?[]const u8 {
 }
 
 fn to_case(comptime input: []const u8, comptime case: enum { camel, pascal }) []const u8 {
-    comptime {
```

Comment:
Why do we need to return `comptime` here? Seems important to signal that this code is intended to be run at comptime.

Reply:
`comptme blk:` has the same effect (the code in the block runs at comptime), but doesn't hit the issue of the compiler thinking `return output[0..len]` happens at runtime (even inside `comptime {}`) for some reason.